### ISSUE TITLE
List rendering bug in Buran 1.12

### ISSUE THREAD

**Issue by acidus99** [Corewala/Buran#37]
hello,

I noticed a render bug in Buran. Consider this Gemtext:

```
* hello I'm a list item!

*I'm a line* that starts with italics!
```

Buran renders both of that as list items, but the Gemtext spec says list lines start with `* ` (`*` and a space) not just *`.

Here is a test page:

gemini://gemi.dev/tests/bug1.gmi

**Comment by Corewala** [comment]
Whyyy is the gemtext spec so inconsistent with spacing rules? Shouldn't be too hard to fix though.

**Comment by acidus99** [comment]
yeah, its crazy. I documented some of the weirdness like this here:

gemini://gemi.dev/weird.gmi

I suspect list lines require a space (e.g. `* foo`) so that lines that maybe have markdown like `*this has emphasis* and this doesn't` won't be rendered as a list.

**Comment by Corewala** [comment]
I guess? Still weird considering that gemini doesn't natively support inline emphasis.
